.gitignore

```diff
@@ -37,3 +37,5 @@ GPATH
 GRTAGS
 GSYMS
 GTAGS
+TODO-INTERNAL
```

Comment:
I've adopted this convention for tracking TODO items on separate feature branches. Let me know if this feels appropriate.